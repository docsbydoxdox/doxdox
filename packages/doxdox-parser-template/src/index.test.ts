import { parseString } from './index';

describe('template parser', () => {
    it('parse empty string', async () => {
        await expect(parseString('test.js', '')).resolves.toEqual(
            expect.objectContaining({
                path: 'test.js',
                methods: []
            })
        );
    });
});
