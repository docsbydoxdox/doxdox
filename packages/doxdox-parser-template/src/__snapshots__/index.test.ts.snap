// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template parser parse empty file 1`] = `
{
  "methods": [],
  "path": "./test/mocks/empty.js",
}
`;

exports[`template parser parse example jsdoc headers (declaration methods) 1`] = `
{
  "methods": [],
  "path": "./test/mocks/declaration.js",
}
`;

exports[`template parser parse example jsdoc headers (function methods) 1`] = `
{
  "methods": [],
  "path": "./test/mocks/function.js",
}
`;
